### PR TITLE
docs: Various cleanup items

### PR DESCRIPTION
**Pull request by rfay**
## The Issue

Docs:

* mention `wsl --shutdown` in the troubleshooting section
* mention add-on extra config with the `docker-compose.*_extra.yaml` technique
* Remove references to obsolete "d9"
* PhpStorm UI no longer uses a telephone, now a "bug"

### CODE REVIEW

File: docs/content/users/extend/additional-services.md [hunk 1]
```diff
@@ -92,6 +92,21 @@ You can update an add-on by running `ddev get <addonname>`, or remove it by runn
...
 If you have add-ons that were installed before v1.22, update them with `ddev get <addonname>` and they will be versioned and available in `ddev get --installed`.
 
+## Adding Custom Configuration to an Add-on
+
+Sometimes it's necessary to add custom configuration to an add-on. For example, in [`ddev-redis-7`](https://github.com/ddev/ddev-redis-7) the `image` is set to `image: redis:7.2-alpine`. If you wanted to change this to use `7.0-alpine` instead, you would have two choices:
+
+1. Remove the `#ddev-generated` line in `docker-compose.redis-7.yaml` and edit the relevant line.
+2. Add a `.ddev/docker-compose.redis-7_extra.yaml` with the contents:
+
+  ```yaml
```
Comment (stasadev) [inline docs/content/users/extend/additional-services.md:102]:
````suggestion
```yaml
````

File: docs/content/users/usage/managing-projects.md [hunk 2]
```diff
@@ -47,62 +47,53 @@ If you’re providing the `settings.php` or `wp-config.php` and DDEV is creating
...
+│          │      │                                           │ or 'root/root'     │
+├──────────┼──────┼───────────────────────────────────────────┼────────────────────┤
+│ Mailpit  │      │ Mailpit: https://drup.ddev.site:8026      │                    │
+│          │      │ `ddev mailpit`                            │                    │
+├──────────┼──────┼───────────────────────────────────────────┼────────────────────┤
+│ All URLs │      │ https://drup.ddev.site,                   │                    │
+│          │      │ https://127.0.0.1:33016,                  │                    │
+│          │      │ http://drup.ddev.site,                    │                    │
+│          │      │ http://127.0.0.1:33017                    │                    │
+└──────────┴──────┴───────────────────────────────────────────┴────────────────────┘
```
Comment (stasadev) [inline docs/content/users/usage/managing-projects.md:96]: It looks a bit broken:

image

Perhaps use `ddev config global --simple-formatting=true` for this output as well?

**Review by stasadev — CHANGES_REQUESTED** [review]

File: docs/content/users/usage/managing-projects.md (hunk 2 above)
Comment (rfay) [reply docs/content/users/usage/managing-projects.md:96]: Thanks, yeah I tried that first and wasn't satisfied with its readability. 

On mine it's pretty much OK, this is Chrome/macOS. I don't greatly like it though. 

image

Comment (rfay) [reply docs/content/users/usage/managing-projects.md:96]: Maybe I should try a screenshot or something.

Comment (rfay) [reply docs/content/users/usage/managing-projects.md:96]: I'm going to go ahead and get this in for now, return to the looks later.

**Pull request merged**